### DISCUSSION TITLE
Suggestion: Option to manage line-in volume from ear trumpet?

### DISCUSSION THREAD

**Discussion by exts** [File-New-Project/EarTrumpet#653] [Ideas]
I have this setup where I have a monitor dedicated for my console (ps4) that I use along side my pc and I have a male to male line-in which I use to listen to that audio. The only way to manage the line-in volume is through the volume control panel or the soundcard control panel. Would be nice to have it if line-in is found to show up like other audio inputs.

My soundcard is: [Sound Blaster Audigy Fx](https://us.creative.com/p/sound-blaster/sound-blaster-audigy-fx) if that's any use.

**Comment by Xanthey** [comment]
I second this. I stream on Twitch and use this software a LOT (and recommend it on my stream.) Really a great tool to adjust multiple software sources on the fly. easily. If i could adjust Line-In as well it would be PERFECT. Line-In is from my Mixer which handles my main mic, ambience mics, and drum kit mics. I can adjust their individual volumes on the mixer but mixing that into the computer audio without having to get up and go to the mixer would be phenomenal.

**Comment by Gimmee99** [comment]
I third this.  I want a super quick way to set the INPUT level per Microphone.  I have a super sensitive Mic and every app sets the level to 100% making it super loud.  Since I cant make a default of 50% STICK for all apps, I had a helper app that used SetChannelVolumeScaler to "fix it" with one double-click.  That stopped working after the April Windows 10 update.
